.eslintrc

```diff
@@ -44,7 +44,8 @@
           "^@swagger-api/apidom-reference/dereference/strategies/openapi-3-1/selectors/uri$",
           "^@swagger-api/apidom-reference/resolve/resolvers/file$",
           "^@swagger-api/apidom-reference/resolve/strategies/openapi-3-1$",
-          "^@swagger-api/apidom-reference/parse/parsers/binary$"
+          "^@swagger-api/apidom-reference/parse/parsers/binary$",
+          "openapi-path-templating"
```

Comment:
I guess the eslint just didn't recognize `exports` field right?

Reply:
No idea, I was getting this error:
`Unable to resolve path to module 'openapi-path-templating'.eslint[import/no-unresolved]`
from:
`/node_modules/openapi-path-templating/types/index"`

Reply:
interesting...